fix: Corrected no-unused-import fix logic and its e2e test config

MaksRomanchuk opened the pull request:
Sorry, accidentally closed the previous PR.
List of changes:
- Fixed e2e test config for no-unused-imports.
- Added script to parent package that mimics e2e-test bash command.
- Implemented import deletion logic so e2e tests pass now.

Updates:
- Added import alias deletion
- Added unused alias import to test files

MaksRomanchuk committed:
fix: Corrected no-unused-import fix logic and its e2e test config

MaksRomanchuk committed:
fix: Fixed unused alias removal

DenisGorbachev commented:
@MaksRomanchuk Please remove `console.log('Node1: ', importedName.node)`

MaksRomanchuk committed:
chore: Removed unnecessary log

MaksRomanchuk commented:
Sorry, fixed

DenisGorbachev commented:
<img width="1310" alt="image" src="https://github.com/user-attachments/assets/23884632-4a5b-4b98-bb83-c3db292ff728" />

@MaksRomanchuk Please fix: if the group becomes empty (all imports removed), then it must remove the whole import line

MaksRomanchuk committed:
fix: Added import group counter update

MaksRomanchuk commented:
Missed the counter update. Should work now

MaksRomanchuk committed:
fix: Fixed overlapping range removal

MaksRomanchuk commented:
@DenisGorbachev 
Updated file content stitching, so now range changes inside already removed range are ignored.
Also, updated comma + separator removal to handle more complex scenarios.
